packages/react-aria-components/docs/FileTrigger.mdx

```diff
@@ -120,6 +120,20 @@ A file trigger can accept multiple files by passsing the `allowsMultiple` proper
 </FileTrigger>
 ```
 
+## Directory Selection
+
+To enable selecting directories instead of files, use the `UNSAFE_directory` property. 
+
+This reflects the webkitdirectory HTML attribute and allows users to select directories and their contents.
```

Comment:
Feel free to make `webkitdirectory` link to https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/webkitdirectory

Reply:
Thanks for the comment. Link added.
[refac: Remove unsafe prefix](https://github.com/adobe/react-spectrum/pull/5444/commits/10b8984dd21818df134a315c507b010d44184a65)